deployment/CP05MOAS-GL379_Deploy.csv

```diff
@@ -34,4 +34,10 @@ TS20190205,,TS20190516,,CP05MOAS-GL379-01-ADCPAM000,6,1,2019-02-05T07:54:00,2019
 TS20190205,,TS20190516,,CP05MOAS-GL379-02-FLORTM000,6,1,2019-02-05T07:54:00,2019-05-16T22:54:03,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-FLORTM-04467,39.9166,-70.36972222,,1000,1000,SS-1
 TS20190205,,TS20190516,,CP05MOAS-GL379-03-CTDGVM000,6,1,2019-02-05T07:54:00,2019-05-16T22:54:03,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-CTDGVM-09363,39.9166,-70.36972222,,1000,1000,SS-1
 TS20190205,,TS20190516,,CP05MOAS-GL379-04-DOSTAM000,6,1,2019-02-05T07:54:00,2019-05-16T22:54:03,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-DOSTAM-00142,39.9166,-70.36972222,,1000,1000,SS-1
-TS20190205,,TS20190516,,CP05MOAS-GL379-05-PARADM000,6,1,2019-02-05T07:54:00,2019-05-16T22:54:03,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50238,39.9166,-70.36972222,,1000,1000,SS-1
\ No newline at end of file
+TS20190205,,TS20190516,,CP05MOAS-GL379-05-PARADM000,6,1,2019-02-05T07:54:00,2019-05-16T22:54:03,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50238,39.9166,-70.36972222,,1000,1000,SS-1
+WJ20190617,,,,CP05MOAS-GL379-00-ENG000000,7,1,2019-06-17T23:14:29,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGCON-GLDRCP-00379,39.9156,-70.9142,,1000,1000,FZ-1
+WJ20190617,,,,CP05MOAS-GL379-01-ADCPAM000,7,1,2019-06-17T23:14:29,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-ADCPAM-49982,39.9156,-70.9142,,1000,1000,FZ-1
+WJ20190617,,,,CP05MOAS-GL379-02-FLORTM000,7,1,2019-06-17T23:14:29,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-FLORTM-05133,39.9156,-70.9142,,1000,1000,FZ-1
+WJ20190617,,,,CP05MOAS-GL379-03-CTDGVM000,7,1,2019-06-17T23:14:29,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-CTDGVM-09453,39.9156,-70.9142,,1000,1000,FZ-1
+WJ20190617,,,,CP05MOAS-GL379-04-DOSTAM000,7,1,2019-06-17T23:14:29,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-DOSTAM-00142,39.9156,-70.9142,,1000,1000,FZ-1
+WJ20190617,,,,CP05MOAS-GL379-05-PARADM000,7,1,2019-06-17T23:14:29,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50277,39.9156,-70.9142,,1000,1000,FZ-1
```

Comment:
Deploy date according to deploy doc 3304-00021-00087 is 2019-06-18 - needs to be corrected in deploy csv (time is correct). Location is correct.
Verified glider s/ns via FCP 3304-00021-00087 (see FCP autoexec for ADCPAM s/n)

Reply:
Fixed